routes/breach-details.js

```diff
@@ -4,10 +4,9 @@ const express = require("express");
 const { getBreachDetail } = require("../controllers/breach-details");
 const { asyncMiddleware } = require("../middleware");
 
-
+// eslint-disable-next-line new-cap
```

Comment:
This seems peppered throughout as well. Not sure if we disable the rule globally (or for the ./routes/ folder only) or see if there are ignore pattern options maybe.